python-libraries/nanover-openmm/src/nanover/openmm/imd.py

```diff
@@ -114,6 +114,7 @@ def __init__(
         self._previous_force_index: Set[int] = set()
         self._frame_index = 0
         self._total_user_energy = 0.0
+        self._user_forces = None
```

Comment:
You'll need something like this to satisfy the type checker:
```suggestion
        self._user_forces: Optional[npt.NDArray] = None
```

Reply:
Another good spot!